tools/ci_build/github/azure-pipelines/linux-gpu-ci-pipeline.yml

```diff
@@ -116,34 +116,41 @@ jobs:
 
+  - checkout: self
+    clean: true
+    submodules: none
+
+  - template: templates/get-docker-image-steps.yml
+    parameters:
+      Dockerfile: tools/ci_build/github/linux/docker/Dockerfile.manylinux2014_cuda11
+      Context: tools/ci_build/github/linux/docker
+      DockerBuildArgs: "--network=host --build-arg POLICY=manylinux2014 --build-arg PLATFORM=x86_64 --build-arg BASEIMAGE=nvidia/cuda:11.6.2-cudnn8-devel-centos7 --build-arg DEVTOOLSET_ROOTPATH=/opt/rh/devtoolset-11/root --build-arg PREPEND_PATH=/opt/rh/devtoolset-11/root/usr/bin: --build-arg LD_LIBRARY_PATH_ARG=/opt/rh/devtoolset-11/root/usr/lib64:/opt/rh/devtoolset-11/root/usr/lib:/opt/rh/devtoolset-11/root/usr/lib64/dyninst:/opt/rh/devtoolset-11/root/usr/lib/dyninst:/usr/local/lib64 --build-arg BUILD_UID=$( id -u )"
+      Repository: onnxruntimecuda11build
+
   - task: CmdLine@2
     inputs:
       script: |
-         set -e -x
-         # We assume the machine doesn't have gcc and python development header files
-         sudo rm -f /build /onnxruntime_src
-         sudo ln -s $(Build.SourcesDirectory) /onnxruntime_src
-         python3 -m pip uninstall -y ort-nightly-gpu ort-nightly onnxruntime onnxruntime-gpu onnxruntime-training onnxruntime-directml ort-nightly-directml onnx -qq
-         cp $(Build.SourcesDirectory)/tools/ci_build/github/linux/docker/scripts/manylinux/requirements.txt $(Build.BinariesDirectory)/requirements.txt
-         # Test ORT with the latest ONNX release.
-         sed -i "s/git+http:\/\/github\.com\/onnx\/onnx.*/onnx/" $(Build.BinariesDirectory)/requirements.txt
-         python3 -m pip install -r $(Build.BinariesDirectory)/requirements.txt
-         python3 -m pip install $(Build.BinariesDirectory)/Release/dist/*.whl
-         ln -s /data/models $(Build.BinariesDirectory)
-         cd $(Build.BinariesDirectory)/Release
-         # Restore file permissions
-         xargs -a $(Build.BinariesDirectory)/Release/perms.txt chmod a+x
-         cd $(Build.SourcesDirectory)/java
-         $(Build.SourcesDirectory)/java/gradlew "cmakeCheck" "-DcmakeBuildDir=$(Build.BinariesDirectory)/Release" "-DUSE_CUDA=1"
-         cd /tmp
-         python3 $(Build.SourcesDirectory)/tools/ci_build/build.py \
-              --build_dir $(Build.BinariesDirectory) --cmake_generator Ninja \
-              --config Release --test \
-              --skip_submodule_sync \
-              --build_shared_lib \
-              --parallel \
-              --build_wheel \
-              --enable_onnx_tests --use_cuda --cuda_version=11.6 --cuda_home=/usr/local/cuda-11.6 --cudnn_home=/usr/local/cuda-11.6 \
-              --enable_pybind --build_java --ctest_path ''
+        set -e -x
+        mkdir -p $HOME/.onnx
+        docker run --gpus all --rm \
+          --volume  $(Build.SourcesDirectory):/onnxruntime_src \
+          --volume $(Build.BinariesDirectory)/Release:/build/Release \
+          --volume /data/models:/build/models:ro \
```

Comment:
Also mount /data/onnx to /data/onnx

Reply:
It is referenced at https://github.com/microsoft/onnxruntime/blob/main/onnxruntime/test/providers/cpu/model_tests.cc#L1138

Reply:
shall we mount /data/onnx in https://github.com/microsoft/onnxruntime/blob/c7b27f44866f6139ecea32b5965e90a34fca3b9e/tools/ci_build/github/azure-pipelines/linux-multi-gpu-ci-pipeline.yml#L31-L39 as well